reference/forms/types/options/choice_label.rst.inc

```diff
@@ -48,3 +48,6 @@ If your choice values are objects, then ``choice_label`` can also be a
         'choices_as_values' => true,
         'choice_label' => 'displayName',
     ));
+
+If set to ``false``, all the label tags will be discarded. You can also return
+``false`` from the callable to handle each choice differently.
```

Comment:
I don't really understand what you mean with this second sentence. Do you mean that returning false from the callable allows you to discard only some labels? In that case, I would say something like "If you want to discard only some label tags, you can also return ``false`` from the callable."